app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,47 @@
+class TeamsController < ApplicationController
+  def index
+    @teams = Team.all
+  end
+
+  def show
+    @team = Team.find(params[:id])
+  end
+
+  def edit
+    @team = Team.find(params[:id])
+    p @team
+  end
+
+  def new
+    @team = Team.new
+  end
+
+  def create
+    @team = Team.create(team_params)
```

Comment:
Make sure to seperate create into `new` and `save`. Should look like:
`@team = Team.new(team_params)`
`if @team.save`
`  redirect_to teams_path`
`else`
`  render 'new'`
`end`
This is incase the team isn't valid